Source/WebCore/page/Navigator.cpp

```diff
@@ -81,7 +81,7 @@ Navigator::~Navigator() = default;
 
 String Navigator::appVersion() const
 {
-    auto* frame = this->frame();
+    RefPtr frame = this->frame();
```

Comment:
All functions called on this object are trivial and inlined in the header, so we don’t need to use a smart pointer here. The same applies to the other pointers you are changing in this PR.